cypress/e2e/Test_F2.cy.js

```diff
@@ -0,0 +1,219 @@
+        cy.get('[data-test="add-to-cart-sauce-labs-bolt-t-shirt"]').click()
+        cy.get('.shopping_cart_container').click()
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').should('be.visible')
+        cy.get('.inventory_item_name').contains('Sauce Labs Bolt T-Shirt').should('be.visible')
+    })
+
+    it('ID = 2C1C3 ', () => {
+        cy.get('[data-test="add-to-cart-sauce-labs-bike-light"]').click()
+        cy.reload()
+        cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click()
+        cy.get('[data-test="add-to-cart-sauce-labs-bolt-t-shirt"]').click()
+        cy.get('.shopping_cart_container').click()
+        cy.get('.inventory_item_name').contains('Sauce Labs Bike Light').should('be.visible')
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').should('be.visible')
+        cy.get('.inventory_item_name').contains('Sauce Labs Bolt T-Shirt').should('be.visible')
+
+ //Dodawanie do koszyka z widoku produktu
+
+    it('ID = 2C2C1 ', () => {
+            cy.get('.inventory_item_name').contains('Sauce Labs Backpack').click()
+            cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click()
+            cy.get('.shopping_cart_container').click()
+            cy.get('.inventory_item_name').contains('Sauce Labs Backpack').should('be.visible')
+        })
+
+    })
+
+    it('ID = 2C2C2 ', () => {
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').click()
+        cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click()
+        cy.get('[data-test="back-to-products"]').click()
+        cy.get('.inventory_item_name').contains('Sauce Labs Bike Light').click()
+        cy.get('[data-test="add-to-cart-sauce-labs-bike-light"]').click()
+        cy.get('.shopping_cart_container').click()
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').should('be.visible')
+        cy.get('.inventory_item_name').contains('Sauce Labs Bike Light').should('be.visible')
+    })
+
+    it('ID = 2C2C3 ', () => {
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').click()
+        cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click()
+        cy.get('.bm-burger-button').click()
+        cy.get('#inventory_sidebar_link').click().then(() => {
+            cy.get('.inventory_item_name').contains('Sauce Labs Bike Light').click()
+        })
+        cy.get('[data-test="add-to-cart-sauce-labs-bike-light"]').click()
+        cy.get('.shopping_cart_container').click()
+        cy.get('.inventory_item_name').contains('Sauce Labs Backpack').should('be.visible')
+        cy.get('.inventory_item_name').contains('Sauce Labs Bike Light').should('be.visible')
+
+    })
+
+    function dodajProd1() {
+        cy.get('[data-test="add-to-cart-sauce-labs-backpack"]').click()
+    }
+    function dodajProd2() {
+        cy.get('[data-test="add-to-cart-sauce-labs-bike-light"]').click()
+    }
```

Comment:
możesz to scalić w jedną funkcję, która przyjmuje stringa i na podstawie tego parametru klika w wybrany produkt